limitador/src/storage/distributed/mod.rs

```diff
@@ -333,19 +247,17 @@ impl CrInMemoryStorage {
 
     fn increment_counter(
         &self,
-        key: Counter,
-        counter: &CrCounterValue<String>,
+        counter: &Counter,
+        store_key: Vec<u8>,
+        store_value: &CrCounterValue<String>,
         delta: u64,
         when: SystemTime,
     ) {
-        counter.inc_at(delta, key.window(), when);
-        let counter = counter.clone();
-        let (expiry, values) = counter.into_inner();
-        let key: CounterKey = key.into();
-        let key = postcard::to_stdvec(&key).unwrap();
+        store_value.inc_at(delta, counter.window(), when);
 
+        let (expiry, values) = store_value.clone().into_inner();
         self.broker.publish(CounterUpdate {
-            key,
+            key: store_key,
```

Comment:
Ah sorry, I had done a refactor of renaming CrCounterValue to StoreValue to make it more clear that is the value type that the distributed store works with, and then I reverted it to avoid the PR diff from being too large.  I forgot to revert the variable names too.